src/runtime/ManagedTypes.cd

```diff
@@ -1,196 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is there something wrong with this file? It is useful when you need to wrap your head around the hierarchy of `ManagedType`.

Reply:
Nope, I recovered it now.